### PR TITLE
add delDel

### PR DESCRIPTION
**Pull request by Tatenda-D-Mawoneke**

### CODE REVIEW

File: Warmup-1/delDel/test/index.test.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// Given a string, if the string "del" appears starting at index 1, return a string where that "del" has been deleted. Otherwise, return the string unchanged.
+
+import { assert } from "chai";
+import { delDel } from "../src";
+
+// delDel("adelbc") → "abc"
+// delDel("adelHello") → "aHello"
+// delDel("adedbc") → "adedbc"
+
+describe("delDel", () => {
```
Comment (JustinGuedes) [inline Warmup-1/delDel/test/index.test.ts:10]: Please add a test for when the input is `delabc` and `abcdel`

**Review by JustinGuedes — CHANGES_REQUESTED** [review]

File: Warmup-1/delDel/test/index.test.ts (hunk 1 above)
Comment (Tatenda-D-Mawoneke) [reply Warmup-1/delDel/test/index.test.ts:10]: Sure.

File: Warmup-1/delDel/test/index.test.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  it("should return return 'adedbc' when passed 'adedbc' keeping the string unchanged", () => {
+    const result = delDel("adedbc");
+    assert.equal(result, "adedbc");
+  });
+  it("should return return 'delabc' when passed 'delabc' keeping the string unchanged", () => {
+    const result = delDel("delabc");
+    assert.equal(result, "delabc");
+  });
+  it("should return return 'abc' when passed 'abcdel'  removing the del", () => {
+    const result = delDel("abcdel");
```
Comment (JustinGuedes) [inline Warmup-1/delDel/test/index.test.ts:28]: Please reread the problem

**Review by JustinGuedes — CHANGES_REQUESTED** [review]

File: Warmup-1/delDel/test/index.test.ts (hunk 2 above)
Comment (Tatenda-D-Mawoneke) [reply Warmup-1/delDel/test/index.test.ts:28]: Thank you for your comment. Having further re-examined the proposed solution, I still do not see what is missing.

> Please advise

File: Warmup-1/delDel/test/index.test.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    assert.equal(result, "abc");
```
Comment (JustinGuedes) [inline Warmup-1/delDel/test/index.test.ts:29]: This is incorrect, please read the problem again.

**Review by JustinGuedes — CHANGES_REQUESTED** [review]

File: Warmup-1/delDel/test/index.test.ts (hunk 3 above)
Comment (Tatenda-D-Mawoneke) [reply Warmup-1/delDel/test/index.test.ts]: Noted, thanks.

**Review by JustinGuedes — CHANGES_REQUESTED** [review]

File: Warmup-1/delDel/src/index.ts [hunk 4]
```diff
@@ -0,0 +1,8 @@
+export function delDel(myString: string): string {
```
[the remark this answers is not in the document]
Comment (Tatenda-D-Mawoneke) [reply Warmup-1/delDel/src/index.ts:1]: Ok.

**Pull request merged**